redbadger/website-honestly
# Client side js

### Motivation

We need client side js to enable us to build more features- specifically the contact us slice.

### Test plan

- view deployment below
- check there is javascript

- works without js
- works cross browser
- works in general
- that navigating around the site works

### Pre-merge checklist

- N/A Unit tests - will be covered by contact us slice/service
  - N/A Design review - backend
